docs/cms-adapters/CMS_ADAPTERS.md

```diff
@@ -158,18 +169,22 @@ As for the other metadata retrieved, a bit of configuration is needed in order t
 
 #### How to install
 
-```bash
+```shell
 yarn ng add @o3r/rules-engine
 ```
+
 or
-```bash
+
+```shell
 npx ng add @o3r/rules-engine
 ```
 
 #### How to use
+
 In _angular.json_ of the app a new build architect needs to be added.
-Here is an example:  
-```JSON
+Here is an example:
+
+```json5
 // angular.json
   ...
```

Comment:
This code area will still be invalid due to the three dots here

Reply:
I think this should not be part of this PR but more on the documentation review.
The purpose of this PR was just to add the language to the `@example` annotations.
Specially this part is a change resulting of a previous comment and was not part of the original PR